res/values/strings.xml

```diff
@@ -376,7 +376,7 @@
     </plurals>
     <!-- The placeholder will be replaced by the name of the recipient in a one-to-one chat. -->
     <string name="chat_new_one_to_one_hint">Send a message to %1$s; they can even receive it without Delta Chat.</string>
-    <string name="chat_new_broadcast_hint">In a broadcast list, others will receive messages in their private chats with you.\n\nThe recipients will not be aware of each other.</string>
+    <string name="chat_new_broadcast_hint">In a broadcast list, the recipients will receive messages in a read-only chat.\n\nThey will not be see the other recipients.</string>
```

Comment:
i think, it should be either "They will not _be seen by_ the other recipients." or  "They will _not see_ the other recipients."

however, to exclude discussions wrt BCC and encryption, maybe just say "In a broadcast list, recipients will receive messages in a read-only chats with you" - i think, this is exactly what is important for the one creating the list. for the recipient things are clear anyway as there is no member list

Reply:
:+1:, just "chat" instead of "chats":

```suggestion
    <string name="chat_new_broadcast_hint">In a broadcast list, recipients will receive messages in a read-only chat with you</string>
```

Reply:
I do think they are the "same" from the view of a user, since they look exactly the same for each recipient (or I'm understanding something wrongly)

Reply:
> I do think they are the "same" from the view of a user

i try to make clear what i was meaning by the following example:

1. the chats are the same form the view of the broadcast-recipients - "Me and the Sender" - broadcast receivers probably won't think much about that anyways (they're soft of passive consumers)

2. but from the view of the broadcast-sender, all recipient's chats are different on the recipient's devices - "RecipientA and Me", "RecipientB and Me", "RecipientC and Me" - which is again different from the sender's chat as "Me and RecipientA, RecipientB and RecipientC"

as the string is shown only in the case of 2., my idea was to use the plural form.

however, maybe this is a weird view and not helpful. as there is so much to explain, it is probably already a bad idea :) as said, it is fine as it is now, no need to change things unless you share this idea :) feel free to ignore my comment completely :)

Reply:
unresolving - the current state does not reflect @Hocuri nor mine suggestion - which are 99% the same beside plural/singular. i would be fine with both

Reply:
i applied the suggestion of @Hocuri - to move forward, but also, maybe the plural of my suggestion opens more questions than it answers, i meanwhile would also slightly prefer it :)